integrations/amazon_sagemaker/src/haystack_integrations/components/generators/amazon_sagemaker/sagemaker.py

```diff
@@ -36,14 +42,16 @@ class SagemakerGenerator:
     ```bash
     export AWS_ACCESS_KEY_ID=<your_access_key_id>
     export AWS_SECRET_ACCESS_KEY=<your_secret_access_key>
+
+    export AWS_SECRET_ACCESS_KEY=<your_secret_access_key>
+
```

Comment:
A bit nitpicky but I don't the need for those lines really. 😅 

```suggestion
    export AWS_SECRET_ACCESS_KEY=<your_secret_access_key>
```